src/Symfony/Component/DependencyInjection/Compiler/MergeExtensionConfigurationPass.php

```diff
@@ -74,7 +79,15 @@ public function process(ContainerBuilder $container)
                     $container->getParameterBag()->mergeEnvPlaceholders($resolvingBag);
                 }
 
+                if ($configAvailable) {
+                    BaseNode::resetPlaceholders();
```

Comment:
already handled in `finally`, isn't it?

Reply:
yep :) though it would miss in case of te-throwing.